docs/api/1.x/index.rst

```diff
@@ -12,18 +12,22 @@ Full detailed API documentation:
    :maxdepth: 2
```

Comment:
This may be too much, one level could help to shrink the full reference TOC
![capture d ecran de 2016-05-23 10-17-10](https://cloud.githubusercontent.com/assets/546692/15464446/aade6dcc-20cf-11e6-9539-d20caa6da896.png)